x/relationships/client/rest/tx.go

```diff
@@ -14,6 +14,8 @@ import (
 func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/relationships", createRelationshipHandler(cliCtx)).Methods("POST")
 	r.HandleFunc("/relationships", deleteRelationshipHandler(cliCtx)).Methods("DELETE")
+	r.HandleFunc("/block", blockUserHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/unblock", unblockUserHandler(cliCtx)).Methods("DELETE")
```

Comment:
I would suggest we rename these endpoints naming both of them `/blacklist`. This way we can `POST` and `DELETE` to a single endpoint instead of two, and we respect the REST guideline of **not** using a verb inside an endpoint. 

```suggestion
	r.HandleFunc("/blacklist", blockUserHandler(cliCtx)).Methods("POST")
	r.HandleFunc("/blacklist", unblockUserHandler(cliCtx)).Methods("DELETE")
```